students/19th/YewonLee/login.js

```diff
@@ -0,0 +1,21 @@
+const loginButton = document.getElementsByClassName("login-button")[0];
+const loginInfo = document.querySelectorAll(".login-info");
+const passwordInput = document.getElementById("password");
+const idInput = document.getElementById("id");
+
+loginInfo.forEach(item => item.addEventListener("keyup", function() {
+    let passwordValue = passwordInput.value;
+    let idValue = idInput.value;
+
+    if(passwordValue !== "" && idValue !== "") {
+        loginButton.removeAttribute("disabled");
+        loginButton.id = "login-botton-enabled";
```

Comment:
```suggestion
        loginButton.id = "login-botton-enabled";
```
 loginButton.id = "login-button-enabled";
같습니다 감사합니다!

Reply:
와 감사합니다!!!!!!!